### PR TITLE
[2021.2] Quality Audit: Material, Camera and Lighting Tooltips

### PR DESCRIPTION
**Pull request by Jonasmortensen**
This PR adds tooltips for the URP material editor, Camera UI and a single light component tooltip.

### CODE REVIEW

**Review by erikabar — APPROVED** [review]
All tooltips look good

**Review by ellioman — APPROVED** [review]

**Review by TheoWong-pixel — APPROVED** [review]

**Review by eh-unity — APPROVED** [review]
LGTM.

File: com.unity.render-pipelines.universal/Editor/ShaderGUI/ParticleGUI.cs [hunk 1]
```diff
@@ -58,7 +58,7 @@ public static class Styles
                 "Controls how visible the distortion effect is. At 0, there’s no visible distortion. At 1, only the distortion effect is visible, not the background.");
 
             public static GUIContent VertexStreams = new GUIContent("Vertex Streams",
```
Comment (alex-vazquez) [inline com.unity.render-pipelines.universal/Editor/ShaderGUI/ParticleGUI.cs:60]:
```suggestion
            public static GUIContent VertexStreams = EditorGUIUtility.TrTextContent("Vertex Streams",
```

File: com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitDetailGUI.cs [hunk 2]
```diff
@@ -8,16 +8,16 @@ internal class LitDetailGUI
...
 
             public static readonly GUIContent detailAlbedoMapText = new GUIContent("Base Map",
-                "Select the texture containing the surface details.");
+                "Select the surface detail texture.The alpha of your texture determines surface hue and intensity.");
 
             public static readonly GUIContent detailNormalMapText = new GUIContent("Normal Map",
-                "Select the texture containing the normal vector data.");
+                "Designates a Normal Map to create the illusion of bumps and dents in the details of this Material's surface.");
 
             public static readonly GUIContent detailAlbedoMapScaleInfo = new GUIContent("Setting the scaling factor to a value other than 1 results in a less performant shader variant.");
```
Comment (alex-vazquez) [inline com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitDetailGUI.cs:22]: Please change them to  `EditorGUIUtility.TrTextContent`

File: com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitGUI.cs [hunk 3]
```diff
@@ -23,7 +23,7 @@ public static class Styles
                 "Select a workflow that fits your textures. Choose between Metallic or Specular.");
 
             public static GUIContent specularMapText =
-                new GUIContent("Specular Map", "Sets and configures the map and color for the Specular workflow.");
+                new GUIContent("Specular Map", "Designates a Specular Map and specular color determining the apperance of reflections on this Material's surface.");
```
Comment (alex-vazquez) [inline com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitGUI.cs:26]: Same here, use  EditorGUIUtility.TrTextContent

File: com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/SimpleLitGUI.cs [hunk 4]
```diff
@@ -22,7 +22,7 @@ public enum SmoothnessMapChannel
         public static class Styles
         {
             public static GUIContent specularMapText =
-                new GUIContent("Specular Map", "Sets and configures a Specular map and color for your Material.");
+                new GUIContent("Specular Map", "Designates a Specular Map and specular color determining the apperance of reflections on this Material's surface.");
```
Comment (alex-vazquez) [inline com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/SimpleLitGUI.cs:25]:
```suggestion
                 EditorGUIUtility.TrTextContent("Specular Map", "Designates a Specular Map and specular color determining the apperance of reflections on this Material's surface.");
```

**Review by alex-vazquez — CHANGES_REQUESTED** [review]
Good PR, just change the `new GUIContent` for `EditorGUIUtility.TrTextContent`

File: com.unity.render-pipelines.universal/Editor/ShaderGUI/ParticleGUI.cs (hunk 1 above)
Comment (Jonasmortensen) [reply com.unity.render-pipelines.universal/Editor/ShaderGUI/ParticleGUI.cs]: Changed! thanks

File: com.unity.render-pipelines.universal/Editor/ShaderGUI/ParticleGUI.cs [hunk 5]
```diff
@@ -178,7 +178,7 @@ public static void FadingOptions(Material material, MaterialEditor materialEdito
                         }
 
                         EditorGUI.indentLevel++;
-                        BaseShaderGUI.TwoFloatSingleLine(new GUIContent("Surface Fade"),
+                        BaseShaderGUI.TwoFloatSingleLine(EditorGUIUtility.TrTextContent("Surface Fade"),
```
Comment (alex-vazquez) [inline com.unity.render-pipelines.universal/Editor/ShaderGUI/ParticleGUI.cs:181]: Can we move this to `Styles`?

**Review by alex-vazquez — CHANGES_REQUESTED** [review]

File: com.unity.render-pipelines.universal/Editor/ShaderGUI/ParticleGUI.cs (hunk 5 above)
Comment (Jonasmortensen) [reply com.unity.render-pipelines.universal/Editor/ShaderGUI/ParticleGUI.cs]: Done!

**Review by alex-vazquez — APPROVED** [review]
Awesome. Good PR!

**Review by oleks-k — APPROVED** [review]
Approving to unblock development. Will create a ticket to do another pass later.

**Pull request merged**